.github/workflows/run_go_tests_k2pg-v2.yaml

```diff
@@ -0,0 +1,21 @@
+name: kafka-to-postgresql-v2-go-tests
+on:
+  pull_request:
+    branches:
+      - main
+      - staging
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
```

Comment:
can we add here the logic that it only runs if somthing was changed e.g., in the umh version? you could check for the installer script if something was changed there

Reply:
It would be *way* easier here to check for changes inside the `deployment/united-manufacturing-hub` folder.

Reply:
ahhh messed up the repos. thought we were in ManagementConsole yes, check for changes in your suggested folder, that is better